### PR TITLE
Support for non-human proteins

### PR DESCRIPTION
**Pull request by bgyori**
This PR adds names and synonyms for proteins of 12 non-human organisms (the ones called "popular organisms" by UniProt) to the grounding resource file. It also allows passing a ranked list of organisms to define a priority order when grounding a string. By default, only human proteins are considered, however, one can also add other organisms to the list as fallback, or provide lists with other organisms being prioritized.

### CODE REVIEW

File: gilda/term.py [hunk 1]
```diff
@@ -18,39 +18,44 @@ class Term(object):
...
             'text': self.text,
             'db': self.db,
             'id': self.id,
             'entry_name': self.entry_name,
             'status': self.status,
-            'source': self.source
+            'source': self.source,
         }
+        if self.organism:
+            js['organism'] = self.organism
```
Comment (cthoyt) [inline gilda/term.py:52]: don't forget to return

File: benchmarks/fplx_evaluation.py [hunk 2]
```diff
@@ -86,7 +96,7 @@ def process_fplx_groundings(df):
             if k == 'PUBCHEM':
                 chebi_id = chebi_client.get_chebi_id_from_pubchem(v)
                 if chebi_id:
-                    grounding['db_refs']['CHEBI'] = 'CHEBI:%s' % chebi_id
```
Comment (cthoyt) [inline benchmarks/fplx_evaluation.py:89]: This makes me very happy 😄

Comment (bgyori) [reply benchmarks/fplx_evaluation.py:89]: Lol yes, it makes me cringe that `chebi_client.get_chebi_id_from_pubchem` used to return a CHEBI ID without the prefix until I changed it.

Comment (cthoyt) [reply benchmarks/fplx_evaluation.py:89]: Oh... then I'm happy for the wrong reason. I thought you were finally coming to the dark side with non-redundant identifiers

**Pull request merged**